docs/docs/faqs/_components/LiquidityMarkets.tsx

```diff
@@ -0,0 +1,243 @@
+import * as React from "react";
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import {
+  CHAIN_ID_TO_CSS_ID,
+  CHAIN_ID_TO_NAME,
+  CHAIN_ID_TO_SYMBOL,
+} from "@site/src/utils/const";
+import TOCInline from "@theme/TOCInline";
+
+const moreAssets = {
```

Comment:
we do not want this, the idea is to reduce the maintenance, we should find a way to automate this